src/safe_apps/serializers.py

```diff
@@ -36,6 +36,9 @@ def to_representation(self, instance: Tag) -> str:
 
 
 class FeatureSerializer(serializers.Serializer[Feature]):
+    class Meta:
+        ref_name = "SafeApp.Feature"
```

Comment:
Similar to the comment above:

```suggestion
        ref_name = "safe_apps.serializers.FeatureSerializer"
```